### PR TITLE
Create project0.html

### PR DESCRIPTION
**Pull request by sayyamgatagat**

### CODE REVIEW

File: project0.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <header>
+        <h1>Check My pic</h1>
+    </header>
+    <div class="Image1">
+        <img src="https://upload.wikimedia.org/wikipedia/commons/1/15/Virat_Kohli_portrait.jpg" alt="Virat_Kohli_portrait" height="500px" width="500px">
+    </div>
+    <footer>
+        <p>Sayyam Gatagat. © 2020</p>
+    </footer>
+    <style>
```
Comment (gvsvarun) [inline project0.html:19]: avoid placing styles at the bottom of the page.
And I think it is better if we move this to a different CSS file

File: project0.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Check My pic</title>
+</head>
+<body>
+    <header>
+        <h1>Check My pic</h1>
+    </header>
+    <div class="Image1">
+        <img src="https://upload.wikimedia.org/wikipedia/commons/1/15/Virat_Kohli_portrait.jpg" alt="Virat_Kohli_portrait" height="500px" width="500px">
```
Comment (gvsvarun) [inline project0.html:14]: let us not use inline styles, move this to a CSS class

File: project0.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    </div>
+    <footer>
+        <p>Sayyam Gatagat. © 2020</p>
```
Comment (gvsvarun) [inline project0.html:17]: nice

File: project0.html (hunk 2 above)
Comment (sayyamgatagat) [reply project0.html:14]: Okay I'll change it

File: project0.html (hunk 1 above)
Comment (sayyamgatagat) [reply project0.html:19]: Will change it

File: project0.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style0.css" type="text/css">    
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (gvsvarun) [inline project0.html:6]: Awesome!

File: project0.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    <title>Check My pic</title>
+</head>
+<body>
+    <header>
+        <h1>Check My pic</h1>
+    </header>
+    <div class="Image1">
```
Comment (gvsvarun) [inline project0.html:13]: Awesome!

**Comment by gvsvarun** [comment]
You can go ahead and merge these changes

**Pull request merged**